src/Symfony/Component/Config/Util/XmlUtils.php

```diff
@@ -18,9 +18,12 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Martin Hasoň <martin.hason@gmail.com>
+ * @author Ole Rößner <ole@roessner.it>
  */
 class XmlUtils
 {
+    const XML_IS_NOT_VALID_MESSAGE = 'The XML is not valid.';
```

Comment:
-1 for defining a public constant with this message. We don't do it anywhere in Symfony